### ISSUE TITLE
thisk4geo.sh redefines K4GEO environment variable

### ISSUE THREAD

**Issue by BrieucF** [key4hep/k4geo#313]
Is it possible to have the script `thisk4geo.sh` redefining the $K4GEO environment variable?

**Comment by giovannimarchiori** [comment]
Hi @BrieucF 
I see that thisk4geo.sh does not (yet) set K4GEO ... and this is done by k4_local_repo inside k4geo instead. Is this request still relevant or somehow superseded by k4_local_repo?

**Comment by BrieucF** [comment]
Yes, with `k4_local_repo` this became less needed but I think it still makes sense to have `thisk4geo.sh` setting K4GEO (and LCGEO for consistency)

**Comment by jmcarcell** [comment]
In this case this is coming from DD4hep: https://github.com/key4hep/k4geo/blob/main/CMakeLists.txt#L152 so it's not completely trivial to do.
